frontend/src/Frontend/UI/TabBar.hs

```diff
@@ -62,6 +64,17 @@ data TabBarCfg t m tab = TabBarCfg
 
 makePactLenses ''TabBarCfg
 
+-- | Tab bar configuration, with dynamic tabs
+data TabBarDynCfg t m tab = TabBarDynCfg
```

Comment:
Instead of duplicating this stuff, can we just use `TabBarDynCfg` for everything and use `constDyn` to retrofit the current users of TabBarCfg with the new dyn config?

Reply:
Done.